### DISCUSSION TITLE
Unable to run project after npm reinstalled

### DISCUSSION THREAD

**Discussion by JulianHSC** [microsoft/react-native-windows#12967] [Q&A] [answered]
I originally got this:
> Task :react-native-safe-area-context:compileDebugKotlin FAILED
77 actionable tasks: 18 executed, 59 up-to-date

I thought that my project was a problem. I uninstalled npm, yarn and reinstalled them again. Now i get this same error. The details are here:

npx react-native run-android
info Installing the app...

> Task :react-native-safe-area-context:compileDebugKotlin FAILED
77 actionable tasks: 18 executed, 59 up-to-date

info 💡 Tip: Make sure that you have set up your development environment correctly, by running npx react-native doctor. To read more about doctor command visit: https://github.com/react-native-community/cli/blob/main/packages/cli-doctor/README.md#doctor

e: file:///C:/xxx/RN_Header/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaContextModule.kt:17:63 Type mismatch: inferred type is Map<String, Any>? but Any was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-safe-area-context:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 9s
error Failed to install the app. Command failed with exit code 1: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081
e: file:///C:/xxx/RN_Header/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaContextModule.kt:17:63 Type mismatch: inferred type is Map<String, Any>? but Any was expected FAILURE: Build failed with an exception. * What went wrong: Execution failed for task ':react-native-safe-area-context:compileDebugKotlin'. > A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction > Compilation error. See log for more details * Try: > Run with --stacktrace option to get the stack trace. > Run with --info or --debug option to get more log output. > Run with --scan to get full insights. > Get more help at https://help.gradle.org. BUILD FAILED in 9s.
info Run CLI with --verbose flag for more details.

**Comment by JulianHSC** [comment]
I have tried by changing the file back to the previous yarn version and it works well. There's obviously a problem with the current version. Please fix it. Case solved.

**Comment by JulianHSC** [comment]
Reopen as issue not solved.

**Reply by chrisglein** [reply, marked as answer]
As mentioned on many other threads, this is the support repo for building an app targeting Windows. You're running commands to build for Android:
`npx react-native run-android`
Ask your questions in the RN core repo.

**Comment by JulianHSC** [comment]
Can you help? Where's the website or link? I don't see it. I found what might be in discord.

**Reply by chrisglein** [reply]
https://github.com/facebook/react-native

**Comment by JulianHSC** [comment]
Thanks.
